cypress/component/Forms/forms.spec.js

```diff
@@ -314,7 +423,7 @@ describe('FormField - Tests', () => {
           'Analytical', 'Prospective', 'Retrospective',
           'Case report', 'Case series', 'Cross-sectional',
           'Cohort study'
-        ]
+        ].map((opt) => {return {displayName: opt, displayValue: opt};})
```

Comment:
I kind of want to keep this version without requiring the developer to provide the object, and allow them to provide just the string, since I think it's probably going to be the most common way to do this